network-api/networkapi/wagtailpages/templates/wagtailpages/fragments/blog-card.html

```diff
@@ -12,18 +12,16 @@
 each blog page's dominant tag)
 
 {% block tags %}
-  {% if hide_classifiers != True %}
-    {% if filtered %}
-      <ul class="list-unstyled mt-3 mb-0">
-      {% for tag in page.specific.tags.all %}
-        <li class="d-inline-block text-capitalize">
-          <a href="{% routablepageurl root.specific "entries_by_tag" tag.slug %}">{{ tag }}</a>{% if not forloop.last %}, {% endif %}
-        </li>
-      {% endfor %}
-      </ul>
-    {% else %}
-      {# Show the tag's own "main" tag (i.e. first tag in its tag list) as each blog card's visible tag #}
-
+  {% if filtered and root %}
```

Comment:
this needs that `{% if hide_classifiers != True %}.....{% endif %}` put back around it, as category pages shouldn't be showing the primary tag or any tags at all.

Reply:
just fixed the conflict.